R/vdiagram-render.R

```diff
@@ -51,17 +51,32 @@ createVdiagram <- function(data) {
   
   vdiagramData <- parseVDiagramData(data)
   
-  vplot <- gsplot(mar=c(7, 3, 4, 2), yaxs = "r", xaxs = "r") %>%
-    points(NA,NA, ylab=styles$plot$ylab, xlab=styles$plot$xlab)
+  vplot <- gsplot(mar = c(7, 3, 4, 2), yaxs = "r", xaxs = "r") %>%
+    points(NA, NA, ylab = styles$plot$ylab, xlab = styles$plot$xlab)
   
   vplot <- do.call(grid, append(list(object=vplot), styles$grid))
   vplot <- do.call(axis, append(list(object=vplot), styles$axis))
-  vplot <- do.call(abline, append(list(object=vplot, a=vdiagramData$maxStage), styles$maxStageLine))
-  vplot <- do.call(abline, append(list(object=vplot, a=vdiagramData$minStage), styles$minStageLine))
+  vplot <-
```

Comment:
Auto-formatted with RStudio, didn't like how that looked, so I cleaned it up a bit. Don't mind much if it's set back the way it was.